features/daemon.feature

```diff
@@ -61,16 +61,16 @@ Feature: Pro Upgrade Daemon only runs in environments where necessary
     Scenario Outline: daemon should run when appropriate on gcp generic lts
         Given a `<release>` machine with ubuntu-advantage-tools installed
         # verify its enabled, but stops itself when not configured to poll
-        When I run `cat /var/log/ubuntu-advantage-daemon.log` with sudo
-        Then stdout matches regexp:
+        When I run `journalctl -o cat -u ubuntu-advantage.service` with sudo
```

Comment:
Why do we need the `cat`?
```suggestion
        When I run `journalctl -u ubuntu-advantage.service` with sudo
```

Reply:
:black_cat:

Reply:
Honestly, I just copied it from [unattached_commands.feature](https://github.com/canonical/ubuntu-pro-client/blob/ef9fe5d042abb52435a2147fa285e3b655150887/features/unattached_commands.feature#L406)

Reply:
I just looked up the option in `man journalctl` and it seems reasonable to use